frontend/src/report/report-page/encoding/Encoding.svelte

```diff
@@ -1,53 +1,110 @@
 <script lang="ts">
 	import { report, reports } from "../../../stores";
-	import SlicesEncoding from "./SlicesEncoding.svelte";
-	import MetricsEncoding from "./MetricsEncoding.svelte";
-	import ModelsEncoding from "./ModelsEncoding.svelte";
+	import SlicesEncodingDropdown from "./SlicesEncodingDropdown.svelte";
+	import SlicesEncodingMultiChoice from "./SlicesEncodingMultiChoice.svelte";
+	import SecondSlicesEncoding from "./SecondSlicesEncoding.svelte";
+	import MetricsEncodingDropdown from "./MetricsEncodingDropdown.svelte";
+	import MetricsEncodingMultiChoice from "./MetricsEncodingMultiChoice.svelte";
+	import ModelsEncodingDropdown from "./ModelsEncodingDropdown.svelte";
+	import ModelsEncodingMultiChoice from "./ModelsEncodingMultiChoice.svelte";
 	import { ChartType } from "../../../zenoservice";
 	import Svelecte from "svelecte";
 
 	const EncodingMap = {
-		slices: SlicesEncoding,
-		models: ModelsEncoding,
-		metrics: MetricsEncoding,
+		slices: {
+			fixed: SlicesEncodingDropdown,
+			multi: SlicesEncodingMultiChoice,
+		},
+		metrics: {
+			fixed: MetricsEncodingDropdown,
+			multi: MetricsEncodingMultiChoice,
+		},
+		models: {
+			fixed: ModelsEncodingDropdown,
+			multi: ModelsEncodingMultiChoice,
+		},
+	};
+	const labelMap = {
+		[ChartType.BAR]: {
+			x: "x",
+			y: "y",
+			color: "color",
```

Comment:
don't change this if it's too much effort, but might make sense to make "color" into "z" instead since it varies depending on the chart.

Reply:
Hmm yeah this makes more sense to me too.